### ISSUE TITLE
Final Report Peer Review - hm458

### ISSUE THREAD

**Issue by HaobingM** [thEKurz/5741Project_FedReserve#11]
For this project, the team aims to explore how the "Fed" is affecting the economy. The independent variables are "Fed actions", including effective rates, press release, etc. And the dependent variables are asset prices, macroeconomic indicators and interest rates. Different methods are tried, such as OLS, ridge, lasso, random forest and XGBoost.

Things I like:
1. The final paper is well written and well structured, and I especially like the way they approach the problem and the way they introduce the variables and techniques to get/clean/impute data.
2. Although there are a lot of variables to consider, they did a great job in proceeding with the problem. And they came to a conclusion which might be useful for further research and analysis.
3. They talked a lot about train/validate/test split and underfitting vs overfitting and tackled these problems. This makes their results robust and trustworthy.

Things for further improvements:
1. Though the paper is clean and professionally written, some of the pictures (on page 5) are ambiguous because they lack x-axis and y-axis.
2. Though their conclusions can help later research of similar topics to a great extent, I really hope they talk more about business-related application about their findings.
3. When introducing methods that we didn't learn from classes (such as NLP, XGBoost), I think it is better to give more detailed introduction and how they would differ from techniques we already learned.

Overall, a very interesting data analysis project and their findings are trustworthy and helpful for further analysis.
